deletefb/deletefb.py

```diff
@@ -1,54 +1,57 @@
 #! /usr/bin/env python
 
+import argparse
+import time
+
 from seleniumrequests import Chrome
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.chrome.options import Options
-import time
-import argparse
+
 
 MAX_POSTS = 5000
 
 def run_delete():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-E",
-                        "--email",
-                        default=None,
-                        help="Your email address associated with the account")
-
-    parser.add_argument("-P",
-                        "--password",
-                        default=None,
-                        help="Your Facebook password")
-
-    parser.add_argument("-U",
-                        "--profile-url",
-                        default=None,
-                        help="""
-                        The link to your Facebook profile, e.g. https://www.facebook.com/your.name
-                        """)
+    parser.add_argument(
+        "-E",
+        "--email",
+        required=True,
+        help="Your email address associated with the account"
+    )
+
+    parser.add_argument(
+        "-P",
+        "--password",
+        required=True,
+        help="Your Facebook password"
+    )
+
+    parser.add_argument(
+        "-U",
+        "--profile-url",
+        required=True,
+        help="The link to your Facebook profile, e.g. https://www.facebook.com/your.name"
+    )
 
     args = parser.parse_args()
 
-    delete_posts(user_email_address=args.email,
-                 user_password=args.password,
-                 user_profile_url=args.profile_url)
+    delete_posts(
+        user_email_address=args.email,
+        user_password=args.password,
+        user_profile_url=args.profile_url
+    )
 
-def delete_posts(user_email_address=None,
-                 user_password=None,
-                 user_profile_url=None):
+def delete_posts(user_email_address,
+                 user_password,
+                 user_profile_url):
     """
-    user_email_address: Your Email
-    user_password: Your password
-    user_profile_url: Your profile URL
+    user_email_address: str Your Email
+    user_password: str Your password
+    user_profile_url: str Your profile URL
     """
-
-    assert all((user_email_address,
-                user_password,
-                user_profile_url)), "Insufficient arguments provided"
-
     # The Chrome driver is required because Gecko was having issues
     chrome_options = Options()
-    prefs = {"profile.default_content_setting_values.notifications" : 2}
+    prefs = {"profile.default_content_setting_values.notifications": 2, 'disk-cache-size': 4096}
```

Comment:
I'm assuming the `disk-cache-size` is necessary to persist sessions across runs? That's useful.